### DISCUSSION TITLE
No Sound when played - Safari

### DISCUSSION THREAD

**Discussion by GrahamEnigma** [mcguirepr89/BirdNET-Pi#607] [General]
I have an established BirdPi that seems to be detecting and feeding BirdWeather.

However, when I connect via web browser and see detections in the browser, pressing the play button in the most recent graphic or 5 Most Recent I can barely hear the audio.  Is there a way to boost the volume of detections?

How can I find the location that the detected audio files are stored?

Cheers

**Comment by ehpersonal38** [comment]
HI @GrahamGIGA 

This wiki article should help you boost the volume of your recordings: https://github.com/mcguirepr89/BirdNET-Pi/wiki/Adjusting-your-sound-card

Detected audio files are stored in `~/BirdSongs/Extracted/By_Date/`

Hope that helps! 🐤

**Comment by GrahamEnigma** [comment]
I'll give it a try, much appreciated!!

Cheers
